packages/app/src/runner/reporter.ts

```diff
@@ -11,12 +11,6 @@ export function setInitializedReporter (val: boolean) {
   hasInitializeReporter = val
 }
 
-async function unmountReporter () {
```

Comment:
this isn't used anywhere. Figured a good time to delete dead code as to unmount in react 18 we would need a root reference and I would rather avoid the complexity